### ISSUE TITLE
New Chapter: Let Engagement Lead the Way

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization-ambassadors/open-org-leaders-manual#9]
This is a proposed chapter from Chad Sansing, Sarah Allen, and Stephanie Wright.

**Comment by chadsansing** [comment]
@semioticrobotic - [a draft copy](https://docs.google.com/document/d/1NZu-btOLKO9KwFDnpuY0p7UTFqVCUR3mpD0m8GAhNnI/edit?usp=sharing
), also sent to you via email.

**Comment by semioticrobotic** [comment]
This is a wonderful chapter, @chadsansing—truly outstanding and the perfect fit for this book. I'm delighted you submitted it.

I've made a few light edits for style and consistency. My most significant edits occurred up front with the introduction, where I suggested reshuffling some paragraphs (take or leave!). Otherwise, I think my comments are pretty self-explanatory and I doubt you'll need to invest too much more time in a second round of edits.

Check it out when you have a moment and we'll chat.
